<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>promptForm</class>
 <widget class="QWidget" name="promptForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>772</width>
    <height>422</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>772</width>
    <height>422</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>772</width>
    <height>422</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Puzzle stacker</string>
  </property>
  <widget class="QLabel" name="good_image">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>70</y>
     <width>315</width>
     <height>215</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>315</width>
     <height>215</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>315</width>
     <height>215</height>
    </size>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;img src=&quot;:/newPrefix/logo.jpg&quot;/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="bad_image">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>70</y>
     <width>315</width>
     <height>215</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>315</width>
     <height>215</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>315</width>
     <height>215</height>
    </size>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;img src=&quot;:/newPrefix/logo.jpg&quot;/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="good_label">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>330</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;Dobre rezultaty&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="bad_label">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>330</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;ZÅ‚e rezultaty&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="prompt_image.qrc"/>
 </resources>
 <connections/>
</ui>
